### ISSUE TITLE
Add terminal to application codes

### ISSUE THREAD

**Issue by lf-** [terminalguide/terminalguide#18]
An example of this set of codes is what happens for meta-arrow-key, as written down in the reviews for https://github.com/troglobit/editline/pull/70.

I noticed this was missing when linking this site there (it is my usual go-to for terminal documentation), but it seems that it is missing documentation for the terminal to application direction, only documenting application to terminal.

**Comment by textshell** [comment]
Yes, unless it's related to a documented application to terminal sequence it currently is not documented.

This is mostly, because i don't feel that this fits very well into a table format.

I actually collected a bunch of data for key press sequences for developing [termpaint](https://termpaint.namepad.de), but got stuck in getting that nicely visualized for this site.

The original idea was to so something fancy with javascript based on a keyboard (because of course these sequences can be keyboard layout specific). But my vanilla javascript isn't that great.

So this is currently pretty much in a "help wanted" state for finding a nice way to present it here.

If we have a nice way to present it, it is just a bunch "setting the terminals to disable as many terminal internal shortcuts as possible and running termpaint's keyboardcollector".

In the meantime the best public dataset i have is https://github.com/termpaint/termpaint/blob/master/tests/input_tests.json for termpaint's selftests but, it does not have details which terminal emits that code for a given key combination.
